src/overlays/gamestates/ovl_title/z_title.c

```diff
@@ -6,23 +6,209 @@
+    GfxPrint_SetPos(printer, 7, 23);
+    GfxPrint_Printf(printer, "[Creator:%s]", gBuildTeam);
+    GfxPrint_SetPos(printer, 7, 24);
+    GfxPrint_Printf(printer, "[Date:%s]", gBuildDate);
+    g = GfxPrint_Close(printer);
+    GfxPrint_Dtor(printer);
+    *gfxp = g;
 }
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/gamestates/ovl_title/func_80800134.s")
+// Note: In other rom versions this function also updates unk_1D4, coverAlpha, addAlpha, visibleDuration to calculate the fade-in/fade-out + the duration of the n64 logo animation
+void Title_Calc(TitleContext* this)
+{
+    this->exit = 1;
+}
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/gamestates/ovl_title/func_808001C8.s")
+void Title_InitView(TitleContext* this, float x, float y, float z)
+{
+    View* view;
+    Vec3f v1;
+    Vec3f v2;
+    Vec3f v3;
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/gamestates/ovl_title/func_8080073C.s")
+    view = &this->view;
+    v3.z = 0;
+    v3.x = 0;
+    v2.z = 0;
+    v2.y = 0;
+    v2.x = 0;
+    v1.x = x;
+    v1.y = y;
+    v1.z = z;
+    v3.y = 1.0;
 
-void func_80800858(TitleContext* this)
+    func_800AA460(view, 30.0f, 10.0f, 12800.0f);
+    func_800AA358(view, &v1, &v2, &v3);
+    func_800AAA50(view, 0xF);
+}
+
+void Title_Draw(TitleContext* this)
 {
-    func_800A9AD0(this, &this->unk_1D0);
+    static s16 sTitleRotY = 0;
+    static u32 D_808009A4 = 0;
+    static Lights1 sTitleLights = gdSPDefLights1(0x64, 0x64, 0x64, 0xFF, 0xFF, 0xFF, 0x45, 0x45, 0x45);
+
+    u16 y;
+    u16 idx;
+    char pad1[0x4];
+    Vec3f v3;
+    Vec3f v1;
+    Vec3f v2;
+    char pad2[0x8];
+    GraphicsContext* gfxCtx = this->state.gfxCtx;
+    Gfx* gfxArr[4];
+
+    func_800C6AC4(&gfxArr, this->state.gfxCtx, "../z_title.c", 395);
+
+    v3.x = 69;
+    v3.y = 69;
+    v3.z = 69;
+    v2.x = -4949.148;
+    v2.y = 4002.5417;
+    v1.x = 0;
+    v1.y = 0;
+    v1.z = 0;
+    v2.z = 1119.0837;
+
+    func_8002EABC(&v1, &v2, &v3, this->state.gfxCtx);
+    gSPSetLights1(gfxCtx->polyOpa.p++, sTitleLights);
+    Title_InitView(this, 0, 150.0, 300.0);
+    func_80093D18(this->state.gfxCtx);
+    Matrix_Translate(-53.0, -5.0, 0, 0);
+    Matrix_Scale(1.0, 1.0, 1.0, 1);
+    Matrix_RotateXYZ(0, sTitleRotY, 0, 1);
+
+    gSPMatrix(gfxCtx->polyOpa.p++, Matrix_NewMtx(this->state.gfxCtx, "../z_title.c", 424), G_MTX_LOAD);
+    gSPDisplayList(gfxCtx->polyOpa.p++, &D_01002720);
+    func_800944C4(this->state.gfxCtx);
+    gDPPipeSync(gfxCtx->polyOpa.p++);
+    gDPSetCycleType(gfxCtx->polyOpa.p++, G_CYC_2CYCLE);
+    gDPSetRenderMode(gfxCtx->polyOpa.p++, G_RM_XLU_SURF2, G_RM_OPA_CI | CVG_DST_WRAP);
+    gDPSetCombineLERP(gfxCtx->polyOpa.p++, TEXEL1, PRIMITIVE, ENV_ALPHA, TEXEL0, 0, 0, 0, TEXEL0, PRIMITIVE, ENVIRONMENT, COMBINED, ENVIRONMENT, COMBINED, 0, PRIMITIVE, 0);
+    gDPSetPrimColor(gfxCtx->polyOpa.p++, 0, 0, 170, 255, 255, 255);
+    gDPSetEnvColor(gfxCtx->polyOpa.p++, 0, 0, 255, 128);
+    
+    _gDPLoadTextureBlockTile(gfxCtx->polyOpa.p++,
```

Comment:
I believe this should be `gDPLoadMultiBlock` instead (which afaik is the same but seems more likely considering the "_").